creds.py

```diff
@@ -0,0 +1,11 @@
+# TELEGRAM
+BOT_TOKEN = ""
+CHANNEL_ID = ''
+
+# REDDIT
+client_id = ""
+client_secret = ""
+password = ""
+user_agent = "udemy bot"
```

Comment:
PRAW / Reddit has a strict requirement for the user_agent string, which `"udemy bot"` doesn't meet:

https://praw.readthedocs.io/en/stable/getting_started/quick_start.html

I'd prioritize fixing this, since it may get you IP-banned if your bot causes any problems.